tests/conftest.py

```diff
@@ -1709,6 +1709,10 @@ def rgw_deployments(request):
         label=constants.RGW_APP_LABEL, namespace=config.ENV_DATA["cluster_namespace"]
     )
     if rgw_deployments:
+        # Force-skipping in case of IBM Cloud -
+        # https://github.com/red-hat-storage/ocs-ci/issues/3863
+        if config.ENV_DATA["platform"].lower() == "ibm_cloud":
```

Comment:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L621 we have ibm_cloud platform in constants. Please use this.